docs/whats-new/staging-release-notes.md

```diff
@@ -1,5 +1,17 @@
 # Staging release notes
 
+## December 2023
+
+**06 December 2023**
+
+Frontend version: 1.0.0-20231130.0955 | Backend version: 1.78.2-20231201.1059
+
+| Type      | Change      | Description |
+| --- | --- | --- |
+| **Change** | Get Existing Users API now checks account existence in the source AAD | Previously, the **Get Existing Users API** implementation excluded orphaned accounts that have been terminated in WoG, but not in TechPass. We modified this API to check if an account exists in the source AAD. For members and users from an unsupported directory, it checks TechPass AAD. For guest accounts from a supported directory, it checks the parent directory.
+| **Change** | New SEED status for TechPass users ineligible for SEED | We've introduced a new **Ineligible** status for users who don't qualify for SEED, such as non-native TechPass users not in WoG AAD. Previously, these users were labeled as **Not Required**, which is intended for users without an assigned license.
```

Comment:
```suggestion
| **Change** | New SEED status for TechPass users ineligible for SEED | We've introduced a new **Ineligible** status for users who don't qualify for SEED, such as non-native TechPass users not in WoG Entra ID . Previously, these users were labeled as **Not Required**, which is intended for users without an assigned license.
```

Reply:
what do u mean by non-native TechPass users?

Reply:
> what do u mean by non-native TechPass users?

I got the term from the JIRA story (https://gdsjira.ship.gov.sg/browse/TECHPASS-3564). Maybe @yqyeoh can provide some context? And can you recommend a better term?